source/administration/object-management/transition-objects-to-minio.rst

```diff
@@ -0,0 +1,228 @@
+.. _minio-lifecycle-management-transition-to-minio:
+
+=============================================
+Transition Objects to Remote MinIO Deployment
+=============================================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+The procedure on this page creates a new object lifecycle management rule that transitions objects from a bucket on a primary MinIO deployment to a bucket on a remote MinIO deployment.
+This procedure supports cost-management strategies such as tiering objects from a "hot" MinIO deployment using NVMe storage to a "warm" MinIO deployment using SSD .
+
+.. todo: diagram
+
+Requirements
+------------
+
+Install and Configure ``mc``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This procedure uses :mc:`mc` for performing operations on the MinIO cluster.
+Install :mc:`mc` on a machine with network access to both source and destination
+clusters. See the ``mc`` :ref:`Installation Quickstart <mc-install>` for
+instructions on downloading and installing ``mc``.
+
+Use the :mc:`mc alias` command to create an alias for the source MinIO cluster.
+Alias creation requires specifying an access key for a user on the source and
+destination clusters. The specified users must have :ref:`permissions
+<minio-lifecycle-management-transition-to-minio-permissions>` for configuring and
+applying transition operations.
+
+.. _minio-lifecycle-management-transition-to-minio-permissions:
+
+Required Source MinIO Permissions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+MinIO requires the following permissions scoped to the bucket or buckets 
+for which you are creating lifecycle management rules.
+
+- :policy-action:`s3:PutLifecycleConfiguration`
+- :policy-action:`s3:GetLifecycleConfiguration`
+
+MinIO also requires the following administrative permissions on the cluster
+in which you are creating remote tiers for object transition lifecycle
+management rules:
+
+- :policy-action:`admin:SetTier`
+- :policy-action:`admin:ListTier`
+
+For example, the following policy provides permission for configuring object
+transition lifecycle management rules on any bucket in the cluster:
+
+.. literalinclude:: /extra/examples/LifecycleManagementAdmin.json
+   :language: json
+   :class: copyable
+
+.. _minio-lifecycle-management-transition-to-minio-permissions-remote:
+
+Required Remote MinIO Permissions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Object transition lifecycle management rules require additional permissions
+on the remote storage tier. Specifically, MinIO requires the remote
+tier credentials provide read, write, list, and delete permissions for the
+remote bucket.
+
+For example, the following policy provides the necessary permission
```

Comment:
Maybe clarify the policy exists on the remote cluster:

> For example, the following policy *on the remote cluster* provides the necessary permission ...

Reply:
Done!